src/peft/tuners/lora/model.py

```diff
@@ -349,16 +357,18 @@ def enable_adapter_layers(self):
         self._set_adapter_layers(enabled=True)
 
     def _get_active_adapter(self) -> str:
-        active_adapter = None
+        active_adapters = None
         for module in self.model.modules():
             if isinstance(module, LoraLayer):
-                active_adapter = module.active_adapter
+                active_adapters = module.active_adapters
 
-        if active_adapter is None:
+        if active_adapters is None:
             raise ValueError(
                 "Something went wrong, no active adapter could be found, please report the issue on GitHub"
             )
-        return active_adapter
+        if len(active_adapters) > 1:
+            raise ValueError("Only set a single active adapter. Multiple active adapters is not supported.")
```

Comment:
I wonder: We use `_get_active_adapter` inside of `disable_adapter_layers` and `_mark_only_adapters_as_trainable`. Should these two methods not work, i.e. instead of raising an error here, change those two methods to instead work with multiple active adapters?

Reply:
We use it mainly to check the `bias` value which can only be not `None` when there is only a single adapter. So, this should be fine.

Reply:
and `_mark_only_adapters_as_trainable` is called when adding a new adapter and as such only that adapter needs to be active.

Reply:
Addressed in the latest commit.